/cable_robot/gcu:
  ros__parameters:
    debugInterval: 2e-1  # timer interval for debug info
    controlUpdateInterval: 1e-6  # timer interval for controls update
    # initial position
    x: 0.0
    y: 0.0
    z: 2.0
    areaSideX: 4.0  # x size of side of area
    areaSideY: 4.0  # y size of side of area
    areaSideZ: 4.0  # z size of side of area
    platformSide: 0.4  # size of size of platform
    maxSpeed: 3.0  # maximum speed of platform
    payloadMass: 4.0 # mass of payload
